src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -926,17 +926,23 @@ private static void VerifyModelForOutVarInNotExecutableCode(SemanticModel model,
             }
         }
 
-        private static void AssertInfoForDeclarationExpressionSyntax(SemanticModel model, DeclarationExpressionSyntax decl)
+        private static void AssertInfoForDeclarationExpressionSyntax(SemanticModel model, DeclarationExpressionSyntax decl, bool expectSymbol = true, bool expectType = true)
         {
             var symbolInfo = model.GetSymbolInfo(decl);
-            Assert.Null(symbolInfo.Symbol);
+            if (expectSymbol)
+            {
+                Assert.NotNull(symbolInfo.Symbol);
```

Comment:
>Assert.NotNull(symbolInfo.Symbol); [](start = 16, length = 34)

We should assert the expected symbol not just that it is not null. Perhaps we should pass expected symbol or null instead of the boolean flag.